content/doc/book/system-administration/admin-password-reset-instructions.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+= Reset the Jenkins administrator password
+
+As a system administrator, you can reset the Jenkins admin password.
+In case you have forgotten or lost the password, this page provides instructions on how you can reset the admin password.
+
+video::_VhOMyWDIcY[youtube, width=640, height=360]
+== Reset the administrator password
+
+1. Log in to your Jenkins controller.
+2. Stop the Jenkins process. You may use this command: `systemctl stop jenkins`.
+3. Edit the Jenkins configuration file (`config.xml`) inside your `jenkins/` or `$JENKINS_HOME` directory.
+4. Look for **useSecurity** and change it from *true* to *false* manually.
+5. Save your file and close it.
+6. Restart the Jenkins service to apply your changes. You may use this command: `systemctl start jenkins`. After restarting Jenkins, navigate to your controller and sign in.
+7. On the dashboard, select  *Manage Jenkins* in the navigation pane on the left side of the page.
+8. On the *Manage Jenkins* page, under the *Security* section, select *Configure Global Security*.
+9. Under *Security Realm*, select *Jenkins' own user database* from the dropdown menu. Ensure the option *Allow users to sign up* is unchecked. Click save.
+This redirects you to the Manage Jenkins page.
+10. In **Manage Jenkins** page, click on **People**.
+11. You will see a People screen showing your username IDs. Click on the user ID that you want to change the password.
+12. Click on Configure. Locate the *Password* section to change your password.
```

Comment:
```suggestion
12. Select Configure using the gear icon or the dropdown menu from the User ID.
Locate the *Password* section to change your password.
```
suggestion to be more precise with instruction, provide each sentence its own line